qiskit_ibm_runtime/options/resilience_options.py

```diff
@@ -36,15 +36,22 @@
     "QuarticExtrapolator",
 ]
 
+ZneExtrapolatorType = Literal[
+    "multi_exponential",
+    "single_exponential",
+    "double_exponential",
+    "linear",
+]
```

Comment:
Not sure how you want to handle the `polynomial_degree_k` for 1 <= k <= 7, but you could include the first couple
```suggestion
ZneExtrapolatorType = Literal[
    "exponential",
    "double_exponential",
    "linear",
    "polynomial_degree_1",
    "polynomial_degree_2",
    "polynomial_degree_3",
    "polynomial_degree_4",
]
```